### PR TITLE
Add Storage Documentation

### PR DESCRIPTION
**Pull request by cooperlees**
- Add brief docs explaining bandersnatch backend storage options
- Need to fill in the serving options for S3 + Swift

### CODE REVIEW

File: docs/storage_options.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+aws_secret_access_key = your s3 secret access key
+# Use endpoint_url to indicate custom s3 endpoint e.g. like minio etc.
+endpoint_url = endpoint url
+# Optional manual signature version for compatibility
+signature_version = s3v4 
+```
+
+### Serving your Mirror
+
+Unknown. To be added.
```
Comment (cooperlees) [inline docs/storage_options.md:60]: @LeoQuote - How do we serve the S3 mirror? Can you write a paragraph or so about options here please?

Comment (LeoQuote) [reply docs/storage_options.md:60]: Certainly. Basically it is served with s3 static sites and amazon cdn (optional, you can use cdn provided by other cloud)

Comment (cooperlees) [reply docs/storage_options.md:60]: I'll just put that line for now - But when I test this out I'll need to learn how to drive this (I'm an AWS n00b as I work @ Facebook ...)

**Comment by LeoQuote** [comment]
should I wait for this pr to be merged before I submit another pr to write more details. Or I should just submit a pr merging to this branch ?

**Comment by cooperlees** [comment]
> should I wait for this pr to be merged before I submit another pr to write more details. Or I should just submit a pr merging to this branch ?

I'll merge to make easier. Reviews are hard to come by for me on bandersnatch ... was hoping for one.

**Pull request merged**